stackrox/k8s-istio-cve-pusher
# ROX-13287: Filter out invalid CVEs

Also did the following:
* Updated go version to 1.18
* Use GitHub actions instead of CircleCI
* Allow dry-runs to output to a local file for testing purposes

Testing Performed:

* `go run ./main -dry-run -output-dir=<some dir I made>`
* Download https://install.stackrox.io/scanner/scanner-vuln-updates.zip
* Replace the contents of the istio directory with the one produced from above ^
* Zip it all up
    * Ensure the files are all named the same as before
* Follow the steps [here](https://install.stackrox.io/scanner/scanner-vuln-updates.zip) to upload the dumps
    * Note: Offline mode is NOT required when wanting to upload these dumps, but you can use it if you don't believe me
* Ensure Central is ok
* Verify Central has the new Istio vuln by checking the file in the container
